bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeExecutionListenerEventType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.model.bpmn.instance.zeebe;
+
+/**
+ * Represents the type of event listener, that indicates when EL should be executed: at the start
+ * (beginning) or at the end (completion) of an element's processing.
+ */
+public enum ZeebeExecutionListenerEventType {
+  start,
+  end
```

Comment:
Hey @saig0,

Thank you for highlighting this! I absolutely agree with the enum values naming convention. 
However, for this particular case, the enum is utilized directly by XML converters that associate XML attributes with enum values. Since XML attribute values are written in lowercase, adopting uppercase enum values would trigger a `java.lang.IllegalArgumentException` from the `camunda-xml-model` library:

```
java.lang.IllegalArgumentException: No enum constant io.camunda.zeebe.model.bpmn.instance.zeebe.ZeebeExecutionListenerEventType.start
	at java.base/java.lang.Enum.valueOf(Enum.java:293)
	at org.camunda.bpm.model.xml.impl.type.attribute.EnumAttribute.convertXmlValueToModelValue(EnumAttribute.java:38)
```

Also, I found other enums such as `MessageVisibleKind` and `ParticipantBandKind`, where lowercase enum values are utilized, probably, for similar reasons.

Reply:
Alright. Thank you for explaining it. Make sense. :+1: